.travis.yml

```diff
@@ -1,10 +1,54 @@
 language: java
 
-jdk:
-  - oraclejdk8
-
 sudo: required
 
-install: true
-
-script: ./gradlew clean build -x test -x signArchives
+matrix:
+  include:
+    - language: java
+      install: true
+      os: osx
+      osx_image: xcode9.3
+      before_install:
+        - brew update
+        - brew outdated xctool || brew upgrade xctool
+        - OPENCV4NODEJS_DISABLE_AUTOBUILD=1 npm install -g opencv4nodejs
```

Comment:
I think is is not necessary to install this module since it is going to seriously increase job duration and also requires multiple additional components.